source/release-notes.txt

```diff
@@ -20,6 +20,104 @@ Release Notes for |bi|
 .. expect and we don't break the links on the downloads page.
 .. (DOCS-9670 for context)
 
+.. _bi-2-10-0:
+
+|bi| 2.10.0
+-----------
+
+*Released Mar 15, 2019*
+
+Bug Fixes
+~~~~~~~~~
+
+- Fixes a bug where aggregate functions returned ``NULL`` in dual
+  queries with the ``DUAL`` keyword omitted.
+
+- Fixes a bug where some invalid ``DUAL`` queries did not return an
+  error.
+
+- Fix of a bug that prevented pushdown of some joins to derived tables
+  selecting from ``DUAL``.
+
+- Fixes some correctness bugs in the in-memory evaluation for 
+  :doc:`mysql-mode conversions </reference/type-conversion/>`.
+
+- Fixes a bug where some trig functions (``ascii``, ``acos``, and
+  ``asin``) could return incorrect results due to missing
+  :ref:`type conversions <type-conversion-modes>`.
+
+- Fixes a bug where some subquery comparisons could return incorrect
+  results due to missing
+  :ref:`type conversions <type-conversion-modes>`.
+
+- Fixes a bug where we lost precision while converting a ``Uint64`` to a
+  ``Decimal``.
+
+- Fixes a correctness bug with the ``INSERT`` scalar function's pushdown
+  translation.
+
+- Fixes a bug where unstable sorting caused non-deterministic renaming
+  of colliding collection/field names during schema mapping.
+
+- Fixes a bug where some valid subquery comparisons returned errors.
+
+- Fixes a bug where some valid subquery and tuple comparisons returned
+  errors.
+
+- Fixes a bug where queries using ``xor`` returned errors on MongoDB 3.2
+  because the generated pipelines required features added in MongoDB
+  3.4.
+
+- Fixes a bug that caused kerberos
+  :ref:`authentication <client-authentication>` to fail with MongoDB
+  4.0.
+
+Date and Time Changes
+~~~~~~~~~~~~~~~~~~~~~
+
+This release resolves a number of inconsistencies in the |bi-short|'s
+date and timestamp semantics. Resolving these inconsistencies resulted
+in some changes to user-facing behavior:
+
+- Removed inconsistent support for partial dates.
```

Comment:
The tense here doesn't match the previous section -- is that intentional?

Reply:
Good catch